src/index.css

```diff
@@ -317,3 +317,8 @@ code {
 #code-mirror-override .cm-editor {
   height: 100% !important;
 }
+
+/* Can't use #code-mirror-override here as we're outside of this div */
+.body-bg .cm-diagnosticAction {
+  @apply bg-primary;
+}
```

Comment:
@franknoirot happy to follow a different path here, this is just a first attempt at fixing this.